pty/ptytest/ptytest.go

```diff
@@ -190,30 +204,20 @@ func (p *PTY) WriteLine(str string) {
 
 func (p *PTY) logf(format string, args ...interface{}) {
 	p.t.Helper()
-	logf(p.t, p.name, format, args...)
-}
-
-func (p *PTY) fatalf(reason string, format string, args ...interface{}) {
-	p.t.Helper()
-	fatalf(p.t, p.name, reason, format, args...)
-}
-
-func logf(t *testing.T, name, format string, args ...interface{}) {
-	t.Helper()
 
 	// Match regular logger timestamp format, we seem to be logging in
 	// UTC in other places as well, so match here.
-	t.Logf("%s: %s: %s", time.Now().UTC().Format("2006-01-02 15:04:05.000"), name, fmt.Sprintf(format, args...))
+	p.t.Logf("%s: %s: %s", time.Now().UTC().Format("2006-01-02 15:04:05.000"), p.name, fmt.Sprintf(format, args...))
 }
 
-func fatalf(t *testing.T, name, reason, format string, args ...interface{}) {
-	t.Helper()
+func (p *PTY) fatalf(reason string, format string, args ...interface{}) {
+	p.t.Helper()
 
 	// Ensure the message is part of the normal log stream before
 	// failing the test.
-	logf(t, name, "%s: %s", reason, fmt.Sprintf(format, args...))
+	p.logf("%s: %s", reason, fmt.Sprintf(format, args...))
 
-	require.FailNowf(t, reason, format, args...)
+	require.FailNowf(p.t, reason, format, args...)
```

Comment:
I'm curious, is there a reason why these 2 lists are not merged into one call, `require.FailNowf`?